pages/linux/lsblk.md

```diff
@@ -25,3 +25,7 @@
 - Output info about block-device topology:
 
 `lsblk -t`
+
+- Exclude the devices specified by the comma-separated list of major device numbers:
+
+`lsblk -e {{1, 7}}`
```

Comment:
Are you sure you meant a space here? That seems rather odd - especially without double quotes.

Reply:
Right, fix it.